ci-deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o nounset -o errexit
+set -x
+
+TMP_DCOS_TERRAFORM=$(mktemp -d)
+git clone https://github.com/bernadinm/dcos-terraform.git $TMP_DCOS_TERRAFORM
+cd $TMP_DCOS_TERRAFORM
+make init
+rm -fr $TMP_DCOS_TERRAFORM/$(grep $(echo $GIT_URL | cut -d'/' -f 5 | cut -d'.' -f1) $TMP_DCOS_TERRAFORM/.gitmodules -B1 | grep path | cut -d' ' -f3)
+cp -fr $WORKSPACE $TMP_DCOS_TERRAFORM/$(grep $(echo $GIT_URL | cut -d'/' -f 5 | cut -d'.' -f1) $TMP_DCOS_TERRAFORM/.gitmodules -B1 | grep path | cut -d' ' -f3)
+PROVIDER="$(git diff master --name-only | grep -E '^modules|^example' | cut -d'/' -f2 | grep -vE 'null|template|localfile|.git*' | sort | uniq | xargs)" make tenv | tee -a env_to_run.log
+for i in $(grep -E '_output.*examples' env_to_run.log | sort | uniq | cut -d' ' -f2); do
+  cd $i;
+  eval $(ssh-agent); rm ssh-key.pub; ssh-keygen -t rsa -b 4096 -f ssh-key -P ''; ssh-add ssh-key
+  terraform apply -auto-approve;
+  terraform destroy -auto-approve;
```

Comment:
this will only destroy when the job was sucessfull. Post build actions is what you want to do

Reply:
Thanks @fatz. Its done.